sonar-project.properties

```diff
@@ -10,4 +10,6 @@ sonar.links.ci=https://github.com/EcoExtreML/zampy/actions
 sonar.python.coverage.reportPaths=coverage.xml
 sonar.python.xunit.reportPath=xunit-result.xml
 sonar.python.pylint.reportPaths=pylint-report.txt
-sonar.python.version=3.9, 3.10, 3.11
\ No newline at end of file
+sonar.python.version=3.9, 3.10, 3.11
+# Exclude certain file from coverage
+sonar.cpd.exclusions = src/zampy/datasets/era5_land.py
```

Comment:
I don't think it's a good idea to completely exclude that file from sonarcloud... why use sonarcloud then :grimacing:

Reply:
If there is a lot of overlap between era5 and era5land, why not make era5land a subclass of era5?

Reply:
😂 Ok I realize that I thought in a wrong way. I am thinking now to make ERA5land inherit from ERA5, given the very small difference. Let me try out. Thanks for the advice @BSchilperoort! That helps me think twice about it 👍 !